robotbase/src/main/java/frc/robot/commands/pickup/ShooterIntakeFromSource.java

```diff
@@ -0,0 +1,32 @@
+package frc.robot.commands.pickup;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.Constants;
+import frc.robot.Robot;
+
+public class ShooterIntakeFromSource extends Command{
+    public ShooterIntakeFromSource(){
+        addRequirements(Robot.intake,Robot.shooter,Robot.pivot);
+    }
+
+    @Override
+    public void initialize(){
+        Robot.pivot.setPivotRotations(Constants.SHOOTER_PIVOT.INTAKE_FROM_SOURCE_ROTATIONS);
```

Comment:
This should be done using a parallel command group with commands specifically for each subsystem. We should never directly interact with multiple subsystems in a single command.